### DISCUSSION TITLE
x/y offsets

### DISCUSSION THREAD

**Discussion by jtaylorme** [AndreRenaud/PDFGen#147] [Q&A] [answered]
Am I understanding things correctly in that the x/y offsets start at the bottom of the page?    That is the behavior I am seeing but wanted to make sure I wasn't missing something as it seems odd.   Also, is there any documentation other than the example code?   Thanks for this project.  Very nice. 

Jim

**Comment by AndreRenaud** [comment, marked as answer]
Yes, coordinates are from the bottom left (that's how PDF works internally, so the API works in the same way).
The docs are basically either the example application, or the doxygen docs. You can generate these with `make docs`, and then browse the html ones from `docs/html/index.html` but I've also attached the generated PDF here. 
[pdfgen.pdf](https://github.com/AndreRenaud/PDFGen/files/15082767/pdfgen.pdf)

**Comment by jtaylorme** [comment]
I have just begun my journey into the guts of PDF and didn't know that.   Thank you.  

Also appreciate the document file.

Jim

**Reply by AndreRenaud** [reply]
No problem. PDFGen is a very minimal library - it's more about simplicity & reduced code than functionality. But that also means that the documentation can be a bit light. Let me know if you get stuck on anything or have any other questions.
